.editorconfig

```diff
@@ -0,0 +1,20 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
+
+# Set default charset and indentation for python files
+[{*.py, coalib/**/*.py, ./coala}]
```

Comment:
Please remove the spaces after `,`.
Check http://editorconfig.org/ for more references.